test/tools/libtesteth/TestSuite.cpp

```diff
@@ -133,49 +113,68 @@ fs::path TestSuite::getFullPath(string const& _testFolder) const
 	return fs::path(test::getTestPath()) / suiteFolder() / _testFolder;
 }
 
-void TestSuite::executeTests(const string& _name, fs::path const& _testPathAppendix, fs::path const& _fillerPathAppendix, std::function<json_spirit::mValue(json_spirit::mValue const&, bool)> doTests) const
+void TestSuite::executeTest(string const& _testFolder, fs::path const& _jsonFileName) const
 {
-	fs::path const testPath = getTestPath() / _testPathAppendix;
+	fs::path boostTestsRepoPath = fs::path(getTestPath());
+	fs::path boostRelativeTestPath = fs::relative(_jsonFileName, boostTestsRepoPath);
+	string testname = _jsonFileName.stem().string();
+	bool isCopySource = false;
```

Comment:
We should also have a `bool isFiller = false;`  Otherwise, we are losing information whether the filename ends in `Filler.json`.